api/rest/v2/observation.md

```diff
@@ -96,10 +100,10 @@ variable.dcids: "Count_Person"
 Request:
 {: .example-box-title}
 
-```bash
+<pre>
```

Comment:
likewise here, i'm noticing the syntax formatting dropped with this change. could we update this back to the triple-backtick with format hints?

Reply:
See comment above.